### ISSUE TITLE
Agenda (16 mei)

### ISSUE THREAD

**Issue by MeMartijn** [MeMartijn/BachelorThesis#8]
## Wat ik deze week af heb gekregen:
- Pipeline gemaakt voor logres waarbij het model netjes met behulp van de validation set gecorrigeerd wordt;
- Alle embeddings verzameld behalve één;
- Max, min en average pooling resultaten binnen (zie bijlage);
- Functie opgezet voor padding, maar hier heb ik nog wat problemen mee:
   - Sommige embeddings zijn heel erg groot (totale vectorlengte van een paar 100 duizend), waardoor mijn geheugen het niet aankan. Ook al gebruik ik mediaan + 1x de std i.p.v. 2x. 
   - Het gebruiken van de mediaan zoals aangegeven plus één standaarddeviatie geeft altijd netjes rond de 80% van de lengte, maar ik heb geen literatuur kunnen vinden die deze techniek onderbouwt. Hoe zou ik dit het beste kunnen verdedigen in mijn scriptie?

<img width="981" alt="Screenshot 2019-05-16 at 09 59 32" src="https://user-images.githubusercontent.com/12677308/57837556-529f2400-77c3-11e9-8f0c-b8741a32a99d.png">

## Voorstel voor volgende week:
- Volledig afmaken van de bovenstaande punten;
- RQ2 (_How well do neural network architecture classify fake news compared to non-neural classification algorithms?_) afmaken.
